### PR TITLE
docs(gen): lading page getting started cards update

### PR DESCRIPTION
**Pull request by jcirinosclwy**
### Description

Updating the content of 'Getting started' cards of the landing page to feature more beginner-friendly pages.

### CODE REVIEW

File: index.mdx [hunk 1]
```diff
@@ -10,36 +10,36 @@ meta:
...
-## Getting Started
+## Getting started
 
 <Grid>
     <SummaryCard
-        title="Quickstart"
-        icon="rocket"
-        description="Learn how to create, set up and use a Scaleway account in a few steps."
-        label="View Quickstart"
+        title="Take a tour"
```
Comment (nerda-codes) [inline index.mdx:17]:
```suggestion
        title="Take a tour of the Scaleway console"
```

**Review by nerda-codes — APPROVED** [review]

File: index.mdx (hunk 1 above)
Comment (jcirinosclwy) [reply index.mdx:17]: @nerda-codes Too long for the link :/

File: index.mdx [hunk 2]
```diff
@@ -10,36 +10,36 @@ meta:
...
         url="/console/account/quickstart/"
     />
     <SummaryCard
-        title="Concepts"
-        icon="info"
-        description="Core concepts that give you a better understanding of Scaleway."
-        label="View Concepts"
-        url="/console/account/concepts/"
+        title="Understand quotas"
+        icon="organization"
```
Comment (jcirinosclwy) [inline index.mdx:25]:
```suggestion
        icon="privacy"
```

File: index.mdx [hunk 3]
```diff
@@ -10,36 +10,36 @@ meta:
...
-## Getting Started
+## Getting started
 
 <Grid>
     <SummaryCard
-        title="Quickstart"
-        icon="rocket"
-        description="Learn how to create, set up and use a Scaleway account in a few steps."
-        label="View Quickstart"
+        title="Take console tour"
```
Comment (jcirinosclwy) [inline index.mdx:17]:
```suggestion
        title="Take a console tour"
```

**Review by bene2k1 — APPROVED** [review]

**Comment by bene2k1** [comment]
LGTM 🎉

**Pull request merged**